regolith/helpers/u_logurlhelper.py

```diff
@@ -0,0 +1,87 @@
+"""Helper for updating a project's log_url
+
+   Log_urls are the google doc links to a project's Projectum Agenda Log
+"""
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import all_docs_from_collection
+
+TARGET_COLL = "projecta"
+
+def subparser(subpi):
+    subpi.add_argument("projectum_id", help="The ID of the Projectum",
+                       default=None)
+    subpi.add_argument("log_url", help="Google Doc url link to project's Projectum Agenda Log")
+    subpi.add_argument("-n", "--number", help="The id you would like to choose from the listed projectum ids")
+    # Do not delete --database arg
+    subpi.add_argument("-d", "--database",
+                       help="The database that will be updated.  Defaults to "
+                            "first database in the regolithrc.json file."
+                       )
+    return subpi
+
+
+class LogUrlUpdaterHelper(DbHelperBase):
+    """
+    Update a projectum's Log_url, will add a new Log_URL if one doesn't yet exist
+    """
+    # btype must be the same as helper target in helper.py
+    btype = "u_logurl"
+    needed_dbs = [f'{TARGET_COLL}']
+
+    def construct_global_ctx(self):
+        """Constructs the global context"""
+        super().construct_global_ctx()
+        gtx = self.gtx
+        rc = self.rc
+        rc.coll = f"{TARGET_COLL}"
+        if not rc.database:
+            rc.database = rc.databases[0]["name"]
+        gtx[rc.coll] = sorted(
+            all_docs_from_collection(rc.client, rc.coll), key=_id_key
+        )
+        gtx["all_docs_from_collection"] = all_docs_from_collection
+        gtx["float"] = float
+        gtx["str"] = str
+        gtx["zip"] = zip
+
+    def db_updater(self):
+        rc = self.rc
+        key = f"{rc.projectum_id}"
```

Comment:
I think the parser will always return rc.projectum_id as a string so you don't need this?